lighthouse-core/gather/gather-runner.js

```diff
@@ -644,6 +649,11 @@ class GatherRunner {
           await GatherRunner.populateBaseArtifacts(passContext);
           isFirstPass = false;
         }
+
+        // Disable fetcher, in case a gatherer enabled it.
+        // This cleanup should be removed once the only usage of
+        // fetcher (fetching arbitrary URLs) is replaced by new protocol support.
+        await driver.fetcher.disableRequestInterception();
```

Comment:
isn't this going to be done in `disposeDriver` in the next line?

Reply:
this clears it for every pass

Reply:
modify the comment to say that? :)